doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1588,17 +1588,26 @@ Controlling the proof flow
       While :tacn:`pose proof` expects that no existential variables are generated by
       the tactic, :tacn:`epose proof` removes this constraint.
 
-.. tacv:: specialize (@ident {* @term}) {? as @simple_intropattern}
-          specialize @ident with @bindings {? as @simple_intropattern}
-   :name: specialize; _
+.. tacn:: specialize @one_term_with_bindings {? @as_ipat }
+
+   .. todo PR: Where is @ident now?  Need to adapt
 
    This tactic works on local hypothesis :n:`@ident`. The
    premises of this hypothesis (either universal quantifications or
-   non-dependent implications) are instantiated by concrete terms coming either
-   from arguments :n:`{* @term}` or from :ref:`bindings`.
-   In the first form the application to :n:`{* @term}`  can be partial. The
-   first form is equivalent to :n:`assert (@ident := @ident {* @term})`. In the
-   second form, instantiation elements can also be partial. In this case the
+   non-dependent implications) are instantiated by concrete terms from
+   :n:`@one_term_with_bindings`.
+
+   .. todo PR:
+      1) what is "non-dependent implication" above?
```

Comment:
"non-dependent implication" would be the same as "non-dependent arrow" or "non-dependent product", i.e. `A -> B` / `forall _ : A, B`. The parentheses just indicates that all premises (both dependent and non-dependent premises) are considered.

Reply:
I used the equivalent term `non-dependent premise` here.  Right?  Already in the glossary.